Src/ILGPU/IR/Transformations/Optimizer.cs

```diff
@@ -124,6 +124,11 @@ public static class Optimizer
             // Apply final DCE phase in release mode
             if (level > OptimizationLevel.O0)
                 builder.Add(new DeadCodeElimination());
+
+            // Append further backend specific transformations in release mode
```

Comment:
Is this change intented for this PR? It appears unrelated to Group.PopBarrierCount

Reply:
Yes and no; perhaps this change should have been submitted in aother PR However, the OpenCL intrinsic does not appear to compile if the newly added `Group.BarrierPopCount` implementation is not inlined in the context of several AMD drivers. For this reason, I have modified the default backend transformations to include the `Inliner` phase by default (which was previously only enabled for the `PTXBackend`).

Reply:
Ah, I see. Fair enough.